tests/unit/test-loader-test.js

```diff
@@ -1,9 +1,15 @@
-/* globals QUnit */
+/* globals QUnit, require */
 import { module, test } from 'qunit';
 
+var testLoaderModulePath = 'ember-cli-test-loader/test-support/index';
+
+if (Object.keys(requirejs.entries).indexOf(testLoaderModulePath) === -1) {
```

Comment:
seems needless to alloc the array of property keys, only to do `indexOf` across it, if it is already basically a lookup table

`if (requirejs.entries[testLoaderModulePath] !== undefined)`